### PR TITLE
Update changelog for 2.24.0

### PR DESCRIPTION
**Pull request by florayan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Release 2.24.0 01-02-2021
+
+- Respond with HTTP 400 when required headers are omitted in CORS preflight request (#468)
+- Allow backend address override (#464)
+- Add an option to gcsrunner to set Envoy application log path. (#467)
```
Comment (nareddyt) [inline CHANGELOG.md:5]: Remove this one, only used by API Gateway

Comment (florayan) [reply CHANGELOG.md]: Done. Thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# Release 2.24.0 01-02-2021
+
+- Respond with HTTP 400 when required headers are omitted in CORS preflight request (#468)
+- Allow backend address override (#464)
+- Add an option to gcsrunner to set Envoy application log path (#467)
```
Comment (TAOXUY) [inline CHANGELOG.md:5]: Should be


- Respond with HTTP 400 when required headers are omitted in CORS preflight request (#468)
- Allow backend address override (#464)
- Propagate trace ID to correlate access logs and traces (#463)

**Review by TAOXUY — APPROVED** [review]

**Review by nareddyt — APPROVED** [review]

**Comment by florayan** [comment]
Thanks Jason and Teju, Could you help to merge this pull request? I am not authorized to merge it, Thanks!

**Pull request merged**